tests/cucumber/docker/run_docker.sh

```diff
@@ -7,7 +7,7 @@ rm -rf test-harness
 rm -rf tests/cucumber/features
 
 # clone test harness
-git clone --single-branch --branch feature/box-storage https://github.com/algorand/algorand-sdk-testing.git test-harness
```

Comment:
TODO: Revert this branch

Reply:
similar issue as `go-algorand-sdk` ... need to merge in latest from `develop` branch before merging this child PR.